### DISCUSSION TITLE
matplotlib

### DISCUSSION THREAD

**Discussion by TheCodeCache** [flet-dev/flet#3247] [Packages] [answered]
https://pypi.org/project/matplotlib/

**Comment by FeodorFitsner** [comment, marked as answer]
[`matplotlib`](https://pypi.flet.dev/matplotlib) and [other packages](https://pypi.flet.dev/) are available in Flet 0.25.0 pre-release and up for both iOS and Android. [Give it a try](https://flet.dev/blog/flet-new-packaging-pre-release) and let us know how that works.
